lib/html_parser/index.js

```diff
@@ -49,5 +49,25 @@ function parse(htmlInput, prefix, postfix, cb) {
   ], cb);
 }
 
+function parsePurified(htmlInput, prefix, postfix, cb) {
+  // strip prefix and postfix from id and class attributes recursively
+  if (typeof prefix !== 'undefined' || typeof postfix !== 'undefined') {
+    async.whilst(function() {
```

Comment:
I managed to break it by making this while loop run forever. All I did was added the prefix `ugc` to part of the html body that's not in a class name.